src/Microsoft.AspNet.OData/app.config

```diff
@@ -4,7 +4,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-13.0.1.0" newVersion="13.0.1.0" />
+        <bindingRedirect oldVersion="0.0.0.0-13.0.0.0" newVersion="13.0.0.0" />
```

Comment:
Curious about this change...

Reply:
I updated the packages from "Manage Nuget packages" window in visual studio. I wasn't in control of the change.

Reply:
Documentation https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/bindingredirect-element

Reply:
I think the issue is here https://github.com/OData/WebApi/blob/d8d3cea08aae3ae1811f3e2ecba74bf1c7e717f3/src/Microsoft.AspNet.OData/Microsoft.AspNet.OData.csproj#L45
The version in `<Reference>` is `13.0.0.0` but the one in the `<HintPath>` is `13.0.1`

cc @corranrogue9 @xuzhg

Reply:
I think the change is ok.